sale_commission/models/sale_commission.py

```diff
@@ -47,9 +47,10 @@ class SaleCommissionSection(models.Model):
     amount_to = fields.Float(string="To")
     percent = fields.Float(string="Percent", required=True)
 
-    @api.one
+    @api.multi
     @api.constrains('amount_from', 'amount_to')
     def _check_amounts(self):
-        if self.amount_to < self.amount_from:
-            raise exceptions.ValidationError(
-                _("The lower limit cannot be greater than upper one."))
+        for section in self:
```

Comment:
this model isn't sale commission but sale.commission.section, so maybe section is correct.